### ISSUE TITLE
how to get train data

### ISSUE THREAD

**Issue by dndmania** [sharat29ag/CDAL#2]
Hi~
Thanks for your great work. The paper was very good, and I read it well.
I want to know how to make a training data file such as '000007.npy'.
can i get the training data making source?

**Comment by sharat29ag** [comment]
> Hi~
> Thanks for your great work. The paper was very good, and I read it well.
> I want to know how to make a training data file such as '000007.npy'.
> can i get the training data making source?

**Comment by sharat29ag** [comment]
Hi, thank you.
These are features extracted from the pre-trained model, in this case, SSD. If you open this .npy corresponding to an image of PASCAL VOC the shape will be 8732x19, which signifies 8732 regions which are bounding box with corresponding softmax values for 19 classes.
